main.go

```diff
@@ -67,6 +67,11 @@ const (
 	// out-of-service taint strategy (GA from 1.28).
 	minK8sMajorVersionOutOfServiceTaint   = 1
 	minK8sMinorVersionGAOutOfServiceTaint = 28
+	leaderElectionLeaseTimeout            = 15 * time.Second
+	leaderElectionRenewTimeout            = 10 * time.Second
+	leaderElectionRetryTimeout            = 2 * time.Second
+	defaultMinSyncPeriod                  = 10 * time.Minute
+	APIGroupWaitTimeout                   = 10 * time.Second
```

Comment:
I think we should keep this lowercase to avoid exporting it. There is a nice style guide here that explains the different situations: https://google.github.io/styleguide/go/decisions#initialisms
```suggestion
	apiGroupWaitTimeout                   = 10 * time.Second
```